pkg/generate/app/app.go

```diff
@@ -259,6 +259,9 @@ func (r *ImageRef) ImageStream() (*imageapi.ImageStream, error) {
 		ObjectMeta: kapi.ObjectMeta{
 			Name: name,
 		},
+		Spec: imageapi.ImageStreamSpec{
+			Tags: map[string]imageapi.TagReference{r.Tag: {DockerImageReference: r.DockerImageReference.String()}},
```

Comment:
The string value should be "latest" - we don't want require admins to guess what the tag they created was.  There's very little value in generating to the remote one.